Resources/Prototypes/_NF/Entities/Structures/Machines/lathe.yml

```diff
@@ -385,3 +379,440 @@
+      - ClothingBackpackHolding
+      - ClothingBackpackSatchelHolding
+      - ClothingBackpackDuffelHolding
+      - ClothingBackpackMessengerHolding
+      - FlashlightSecliteNF
+    ## Weapons
+      - WeaponGrapplingGun
+      - KatanaNF
+      - KukriKnifeNF
+      - MacheteNF
+      - Stunbaton
+      - BolaNF
+      - ExplosivePayload
+      - FlashPayload
+      - ChemicalPayload
+      - WeaponShotgunKammererNF
+      - WeaponAdvancedLaser # After the changes made to windows shouldn't be that much of an issue now if it's widely available
+      - WeaponAntiqueLaserNF # After the changes made to windows shouldn't be that much of an issue now if it's widely available
+      - WeaponLaserCarbine
+      - WeaponDisabler
+    ## Mags / Speedloaders / Clips
+    ## Ammo boxes
+      - BoxBeanbag
+      - BoxShellTranquilizer
+      - MagazineBoxLightRifleRubber
+      - MagazineBoxMagnumRubber
+      - MagazineBoxPistolRubber
+      - MagazineBoxRifleRubber
+      - MagazineShotgunBeanbag
+
+- type: entity
+  id: MercenaryTechFabHacked
+  parent: [ MercenaryTechFab, BaseHyperlathe ]
+  name: hacked mercenary techfab
+  description: Prints the bare minimum of tools of the trade that any budget mercenary or private security could need. Nothing fancy.
+  components:
+  - type: Sprite
+    sprite: _NF/Structures/Machines/techfab.rsi
+    layers:
+    - state: icon
+      map: ["enum.LatheVisualLayers.IsRunning"]
+    - state: mercenary
+    - state: unlit
+      shader: unshaded
+      map: ["enum.PowerDeviceVisualLayers.Powered"]
+    - state: inserting
+      map: ["enum.MaterialStorageVisualLayers.Inserting"]
+    - state: panel
+      map: ["enum.WiresVisualLayers.MaintenancePanel"]
+    - sprite: _NF/Structures/Holo/graffiti/simple.rsi
+      state: holograffiti01
+      map: [ "graffiti" ]
+      shader: unshaded
+      scale: 0.7, 0.7
+  - type: Machine
+    board: MercenaryTechFabCircuitboardHackedNF
+  - type: MaterialStorage
+    whitelist:
+      tags:
+      - Sheet
+      - RawMaterial
+      - Ingot
+  - type: RandomSprite
+    available:
+    - graffiti:
+        holograffiti01: CyberpunkNeon
+        holograffiti02: CyberpunkNeon
+        holograffiti03: CyberpunkNeon
+        holograffiti04: CyberpunkNeon
+        holograffiti05: CyberpunkNeon
+        holograffiti06: CyberpunkNeon
+        holograffiti07: CyberpunkNeon
+        holograffiti08: CyberpunkNeon
+        holograffiti09: CyberpunkNeon
+        holograffiti10: CyberpunkNeon
+        holograffiti11: CyberpunkNeon
+        holograffiti12: CyberpunkNeon
+        holograffiti13: CyberpunkNeon
+        holograffiti14: CyberpunkNeon
+        holograffiti15: CyberpunkNeon
+        holograffiti16: CyberpunkNeon
+        holograffiti17: CyberpunkNeon
+        holograffiti18: CyberpunkNeon
+        holograffiti19: CyberpunkNeon
+        holograffiti20: CyberpunkNeon
+        holograffiti21: CyberpunkNeon
+        holograffiti22: CyberpunkNeon
```

Comment:
Might be a good idea to move this to some abstract parent (BaseRandomGraffiti?) considering it's on quite a few things.

Reply:
Fair